package/MDAnalysis/visualization/lintools/clock.py

```diff
@@ -0,0 +1,87 @@
+"""Clock Diagrams  --- :mod:`MDAnalysis.visualization.lintools.clock`
+=====================================================================
+
+.. autofunction:: create_clock_diagram
+"""
+
+import _io
```

Comment:
At the risk of sounding silly, what is this? Is it a type annotated version of the io library? Is it stable/meant to be used? The underscore makes it look a little suspect.